packages/jest-watcher/src/lib/Prompt.ts

```diff
@@ -39,7 +39,7 @@ export default class Prompt {
 
   enter(
     onChange: (pattern: string, options: ScrollOptions) => void,
-    onSuccess: () => void,
+    onSuccess: (pattern: string) => void,
     onCancel: () => void,
```

Comment:
this has different fn signature than https://github.com/facebook/jest/pull/7447/files?w=1#diff-048f6a2e1f2d542739b5b37c81a7e9c3R16, I think it's still should be `_onCancel: (value?: string) => void `

Reply:
We initialize to empty string, so the correct type is that it's passed. right?